locations/perleberger36.yml

```diff
@@ -0,0 +1,183 @@
+    username: "root"
+    password: "/root/pwd.txt"
+    daytime_limit: "2-7"
+  - name: " perleberger36-turm-5ghz-o"
+    target: "10.36.243.182"
+    username: "root"
+    password: "/root/pwd.txt"
+    daytime_limit: "2-7"
+  - name: "perleberger36-turm-5ghz-sw"
+    target: "10.36.243.186"
+    username: "root"
+    password: "/root/pwd.txt"
+    daytime_limit: "2-7"
+
+ipv6_prefix: "2001:bf7:750:3300::/56"
+
+networks:
+  - vid: 10
+    role: mesh
+    name: mesh_lan
+    prefix: 10.36.243.192/32
+    ipv6_subprefix: -1
+
+  - vid: 101
+    role: mesh
+    name: mesh_strom2a
+    prefix: 10.36.243.160/30
+    ipv6_subprefix: -2
+    assignments:
+      perleberger36-core: 1
+      perleberger36-strom2a: 2
+
+  - vid: 102
+    role: mesh
+    name: mesh_tu
+    prefix: 10.36.243.164/30
+    ipv6_subprefix: -3
+    assignments:
+      perleberger36-core: 1
+      perleberger36-tu: 2
+
+  - vid: 103
+    role: mesh
+    name: mesh_bht
+    prefix: 10.36.243.168/30
+    ipv6_subprefix: -4
+    assignments:
+      perleberger36-core: 1
+      perleberger36-bht: 2
+
+  - vid: 104
+    role: mesh
+    name: mesh_scherer8
+    prefix: 10.36.243.172/30
+    ipv6_subprefix: -5
+    assignments:
+      perleberger36-core: 1
+      perleberger36-scherer8: 2
+
+  - vid: 201
+    role: mesh
+    name: turm_5ghz_nw
+    prefix: 10.36.243.176/30
+    ipv6_subprefix: -6
+    assignments:
+      perleberger36-core: 1
+      perleberger36-turm-5ghz-nw: 2
+
+  - vid: 202
+    role: mesh
+    name: turm_5ghz_o
+    prefix: 10.36.243.180/30
+    ipv6_subprefix: -7
+    assignments:
+      perleberger36-core: 1
+      perleberger36-turm-5ghz-o: 2
+
+  - vid: 203
+    role: mesh
+    name: turm_5ghz_sw
+    prefix: 10.36.243.184/30
+    ipv6_subprefix: -8
+    assignments:
+      perleberger36-core: 1
+      perleberger36-turm-5ghz-sw: 2
+
+  - vid: 11
+    role: dhcp
+    prefix: 10.36.244.0/24
+    ipv6_subprefix: 0
+    untagged: true
+    inbound_filtering: false
+    enforce_client_isolation: false
+    assignments:
+      perleberger36-core: 1
+
+  - vid: 50
+    role: mgmt
+    prefix: 10.36.243.128/28
+    gateway: 1
+    dns: 1
+    ipv6_subprefix: 1
+    assignments:
+      # 10.36.243.129/32
+      perleberger36-core: 1
+      # 10.36.243.130/32
+      perleberger36-dachboden-erx1: 2
+      # 10.36.243.131/32
+      perleberger36-dachboden-erx2: 3
+      # 10.36.243.132/32
+      perleberger36-turm-erx1: 4
+      # 10.36.243.133/32
+      perleberger36-turm-erx2: 5
+      # 10.36.243.134/32
+      perleberger36-turm-2ghz-nw: 6
+      # 10.36.243.135/32
+      perleberger36-turm-2ghz-no: 7
+      # 10.36.243.136/32
+      perleberger36-turm-2ghz-o: 8
+      # 10.36.243.137/32
+      perleberger36-turm-2ghz-so: 9
+      # 10.36.243.138/32
+      perleberger36-turm-2ghz-sw: 10
+      # 10.36.243.139/32
+      # perleberger36-hof-2ghz-n: 11; offline, but listed in Wiki: https://wiki.freifunk.net/Berlin:Standorte:Heilige-Geist-Kirche_Moabit
```

Comment:
please add hof-2ghz-n.  I will fix it someday soon.  Might as well add the mgmt address now.

Reply:
done